### PR TITLE
DOC: Add SFD for Bookmarks

### PR DESCRIPTION
**Pull request by catong**
https://openedx.atlassian.net/browse/DOC-2489

This PR adds the bookmarks doc to both the partner and open edX versions of the Learners' Guide. Planned for publication when the feature releases - possibly Dec 9?

A separate PR exists for events documentation for this feature. https://github.com/edx/edx-documentation/pull/368

NB: Files were restored from the original limited release of the documentation in June 2015. 
Reviewers please keep an eye out for any implementation/workflow details that changed between then and the coming release that are not reflected in this documentation. Thanks!
### Reviewers
- [x] Subject matter expert: @symbolist 
- [x] Product owner: @explorerleslie 
- [x] Doc team review: @srpearce, or @lamagnifica or @pdesjardins 
### Testing
- [x] Run ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/students/SFD_bookmarks.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.. _SFD Bookmarks:
+
+##############################
+Bookmarks
+##############################
+
+To help you mark content and come back to it later, you can add a bookmark to
+any courseware page. You can see the pages you have bookmarked in an
```
Comment (explorerleslie) [inline en_us/shared/students/SFD_bookmarks.rst]: @catong remind me, do we use units or pages in doc? Technically user bookmarks units, but I'm not sure if pages = units in doc.

File: en_us/shared/students/SFD_bookmarks.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Bookmarks** page. To access the **My Bookmarks** page, select **Bookmarks** in
+the upper left corner of the page.
+
+.. image:: ../../shared/students/Images/SFD_Bkmk_BookmarksButton.png
+ :width: 500
+ :alt: A courseware page with the "Bookmarks" button in the upper left corner
+     and showing several bookmark icons in the learning sequence
+
+The **My Bookmarks** page lists all of your bookmarks in reverse chronological
+order. The page includes the location of the bookmark in the courseware and
```
Comment (explorerleslie) [inline en_us/shared/students/SFD_bookmarks.rst]: would the extra specificity of "reverse chronological order of when you bookmarked them" be helpful? I'm not sure if learners will immediately understand which chronology we're using for bookmarks.

**Comment by explorerleslie** [comment]
@catong review done. Two minor things but I'll leave my :+1: because I don't need to re-review.

File: en_us/shared/students/SFD_bookmarks.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/students/SFD_bookmarks.rst]: @explorerleslie in the learner's guides we prefer "pages"; "unit" is a term that course teams are more familiar with from the Studio point of view. Unless it seems wrong to you, I think it is natural for students to think about pages that they read or work through in a course.

File: en_us/shared/students/SFD_bookmarks.rst (hunk 2 above)
Comment (catong) [inline en_us/shared/students/SFD_bookmarks.rst]: @explorerleslie if there is a defined order I think it's useful to mention it, but I might simplify the wording as "most recently created at the top" or something similar.

File: en_us/shared/students/SFD_bookmarks.rst (hunk 1 above)
Comment (explorerleslie) [inline en_us/shared/students/SFD_bookmarks.rst]: @catong nope, that makes sense -- I just wanted to double check what our standard was.

File: en_us/shared/students/SFD_bookmarks.rst (hunk 2 above)
Comment (explorerleslie) [inline en_us/shared/students/SFD_bookmarks.rst]: @catong oh, yes, I like that wording you put in your comments.

File: en_us/shared/students/SFD_bookmarks.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+.. contents::
+ :local:
+ :depth: 1
+
+***************************
+Add or Remove a Bookmark
+***************************
+
+You can bookmark any page in the courseware. To bookmark a page, select
```
Comment (srpearce) [inline en_us/shared/students/SFD_bookmarks.rst]: courseware -> course

**Comment by srpearce** [comment]
:+1: after the instance of "courseware" is changed.

**Comment by symbolist** [comment]
:+1:

**Comment by catong** [comment]
Squashed commits, tests run clean. Considering doc complete and waiting to merge when the feature releases.

**Comment by explorerleslie** [comment]
@catong sorry, one addition that we made was that there is now going to be a hard limit of 100 bookmarks for any given user. See this task: https://openedx.atlassian.net/browse/TNL-2810. Can you add a little note to the doc?

**Comment by catong** [comment]
@explorerleslie Initially I was going to just let that be managed with messaging in the UI for the few people who will hit 100 bookmarks, but yes, I will add something. Will tag you with the change, thanks.

File: en_us/shared/students/SFD_bookmarks.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
+.. _SFD Bookmarks:
+
+##############################
+Bookmarks
+##############################
+
+To mark course content so that you can come back to it later, you can bookmark
+any course page, up to a limit of 100 bookmarks for one course.
```
Comment (catong) [inline en_us/shared/students/SFD_bookmarks.rst:8]: @explorerleslie Added text in the intro, as well as a note under "Add or Remove a Bookmark" (line 27 below).

Comment (explorerleslie) [inline en_us/shared/students/SFD_bookmarks.rst:8]: :+1:

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/715
